### PR TITLE
Set updatedAt to match createdAt on entity creation

### PR DESCRIPTION
**Pull request by christineyost**
This PR is for [API-15021](https://vajira.max.gov/browse/API-15021). The updatedAt field will now be populated with the current time on entity creation. updatedAt is no longer nullable. This change includes updating existing data in the database migration - if updatedAt is currently null it will be set to the same value as createdAt.

### CODE REVIEW

File: migrations/Migration20220603214843.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20220603214843 extends Migration {
+
+  async up(): Promise<void> {
+    this.addSql('update "render_config" set "updated_at" = "created_at" where "updated_at" is null;');
```
Comment (mheusgit) [inline migrations/Migration20220603214843.ts:6]: That works

File: migrations/Migration20220603214843.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    this.addSql('alter table "render_config" drop constraint if exists "render_config_updated_at_check";');
+    this.addSql('alter table "render_config" alter column "updated_at" type timestamptz(0) using ("updated_at"::timestamptz(0));');
+    this.addSql('alter table "render_config" alter column "updated_at" set not null;');
+
+    this.addSql('update "gateway_config" set "updated_at" = "created_at" where "updated_at" is null;');
+    this.addSql('alter table "gateway_config" drop constraint if exists "gateway_config_updated_at_check";');
```
Comment (mheusgit) [inline migrations/Migration20220603214843.ts:12]: These constraints that were dropped.  I don't see them in the migration files at any point in time

**Review by mheusgit — APPROVED** [review]
Looks good

**Comment by mheusgit** [comment]
Delay this tickets merge until after 16:00 EST since we have a MR

**Review by dsteeve — APPROVED** [review]
looks good!

**Pull request merged**